File: app/src/main/java/com/example/bloodbank/ui/activity/SplashScreenActivity.java
package com.example.bloodbank.ui.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.example.bloodbank.ui.activity.AuthenticationActivity;

public class SplashScreenActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(this, WelcomeActivity.class));
        finish();
    }
}

File: app/src/main/java/com/example/bloodbank/ui/activity/SplashScreenActivity.kt
package com.example.bloodbank.ui.activity

import android.content.Intent
import android.os.Bundle
import com.example.bloodbank.ui.base.BaseActivity

class SplashScreenActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, WelcomeActivity::class.java))
        finish()
    }
}
